Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst

```diff
@@ -132,77 +112,123 @@ Some examples:
 
-   <INCLUDE_TYPOSCRIPT: source="FILE:EXT:my_extension/Configuration/TypoScript/myMenu.typoscript">
+Most usages of :typoscript:`<INCLUDE_TYPOSCRIPT:` can be turned into :typoscript:`@import`
+easily. A few examples:
 
-More details:
+..  code-block:: diff
 
-* Keyword "FILE":
+    -<INCLUDE_TYPOSCRIPT: source="FILE:EXT:my_extension/Configuration/TypoScript/myMenu.typoscript">
+    +@import 'EXT:my_extension/Configuration/TypoScript/myMenu.typoscript'
 
-  A reference to a single file relative to :php:`\TYPO3\CMS\Core\Core\Environment::getPublicPath()`.
+     # Including .typoscript files in a single (non recursive!) directory
+    -<INCLUDE_TYPOSCRIPT: source="DIR:EXT:my_extension/Configuration/TypoScript/" extensions="typoscript">
+    +@import 'EXT:my_extension/Configuration/TypoScript/*.typoscript'
 
-  Paths relative to the including file can be used, if the inclusion is called
-  from inside a file. These paths start with :file:`FILE:./`. This allows simple, nested
-  TypoScript templates that can be moved or copied without the need to adapt all includes.
+    # Including .typoscript and .ts files in a single (non recursive!) directory
+    -<INCLUDE_TYPOSCRIPT: source="DIR:EXT:my_extension/Configuration/TypoScript/" extensions="typoscript,ts">
+    +@import 'EXT:my_extension/Configuration/TypoScript/*.typoscript'
+     # Rename all files ending on .ts to .typoscript
 
-  Files within extensions can be used using :file:`FILE:EXT:my_extension/path/to/file.typoscript`.
-  This should be the preferred usage, importing files not located within extensions is
-  likely to be deprecated with TYPO3 v13, along with the file ending :file:`.txt`.
+     # Including a file conditionally
+    -<INCLUDE_TYPOSCRIPT: source="FILE:EXT:my_extension/Configuration/TypoScript/user.typoscript" condition="[frontend.user.isLoggedIn]">
+    +[frontend.user.isLoggedIn]
+    +    @import 'EXT:my_extension/Configuration/TypoScript/user.typoscript'
+    +[END]
 
-* Keyword "DIR":
+There are a few more use cases that cannot be transitioned so easily since
+:typoscript:`@import` is a bit more restrictive.
 
-  Can be used instead of :typoscript:`FILE` to include multiple files at once.
-  It includes all files from a directory relative to :php:`\TYPO3\CMS\Core\Core\Environment::getPublicPath()`,
-  including subdirectories.
+As one restriction :typoscript:`@import` cannot include files from arbitrary directories
+like :file:`fileadmin/`, but only from extensions by using the :typoscript:`EXT:`
+prefix. Instances that use :typoscript:`<INCLUDE_TYPOSCRIPT:` with :typoscript:`source="FILE:./someDirectory/..."`
+should move this TypoScript into a project or site extension. Such instances are also encouraged to
+look into the TYPO3 v13 "Site sets" feature and eventually transition towards it along the way.
 
-  Files are included in alphabetical. Also files are included first, then directories.
+:typoscript:`@import` allows to import files with the file ending `.typoscript`
```

Comment:
You also mentioned that above in an important section.

Reply:
Repetition cannot harm